src/lib/discord/commando/commands/tracked.js

```diff
@@ -39,6 +39,10 @@ exports.run = async (client, msg, [args]) => {
 		const quests = await client.query.selectAllQuery('quest', { id: target.id })
 		const invasions = await client.query.selectAllQuery('invasion', { id: target.id })
 		const maplink = `https://www.google.com/maps/search/?api=1&query=${human.latitude},${human.longitude}`
+		if (args.includes('area')) {
+			return msg.reply(`You are currently set to receive alarms in ${human.area}`)
+		}
+		if (!args.includes(args[0])) {
```

Comment:
You don't need the second 'if' statement because you are doing a return above.
You could argue that this whole if section could be higher up to avoid the queries.
Should make area translatable:
eg args.includes(client.translator.translate('area'))